.github/workflows/go.yml

```diff
@@ -76,6 +83,7 @@ jobs:
           mkdir --parents /home/runner/go/src/github.com/vertgenlab/ &&
           cp -r ../gonomics /home/runner/go/src/github.com/vertgenlab/ &&
           cd /home/runner/go/src/github.com/vertgenlab/gonomics &&
+          go install ./... &&
           go test -coverprofile=coverage.txt -covermode=atomic ./...
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v2
```

Comment:
```suggestion
        uses: codecov/codecov-action@v3
```

Reply:
codecov uses v3: https://github.com/codecov/codecov-action#migration-from-v1-to-v3